From caf3a4d17dfd2f36f4f2c0661323869695e42fcd Mon Sep 17 00:00:00 2001
From: Chaoyu Tang <ctang@cloudera.com>
Date: Fri, 7 Apr 2017 08:41:46 -0400
Subject: [PATCH 1062/1164] CDH-52461 HIVE-16394: HoS does not support queue
 name change in middle of session (Chaoyu Tang,
 reviewed by Xuefu Zhang, Rui Li)

(cherry picked from commit 72fb816b814ee931d8a2c93e1dddcb9478f3674a)

Change-Id: I5d6f19a09d750a671c3e712c93c11e0a9dccfce2
---
 .../java/org/apache/hadoop/hive/conf/HiveConf.java |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
index 52c193f..d328d7b 100644
--- a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
+++ b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
@@ -2507,6 +2507,9 @@ private boolean isSparkRelatedConfig(String name) {
       }
     } else if (name.startsWith("hive.spark")) { // Remote Spark Context property.
       result = true;
+    } else if (name.equals("mapreduce.job.queuename")) {
+      // a special property starting with mapreduce that we would also like to effect if it changes
+      result = true;
     }
 
     return result;
-- 
1.7.9.5

